src/org/mirah/builtins/builtins.mirah

```diff
@@ -20,25 +20,31 @@ import java.util.Collections
 
 class Builtins
   def self.initialize_builtins(type_system:TypeSystem)
-    type_system.extendClass('java.util.Collection', CollectionExtensions.class)
-    type_system.extendClass('java.util.Map', MapExtensions.class)
-    type_system.extendClass('java.util.List', ListExtensions.class)
-    type_system.extendClass('java.lang.Object', ObjectExtensions.class)
-    type_system.extendClass('java.lang.Iterable', EnumerableExtensions.class)
-    type_system.extendClass('java.lang.Iterable', IterableExtensions.class)
-    type_system.extendClass('java.lang.String', StringExtensions.class)
-    type_system.extendClass('java.lang.StringBuilder', StringBuilderExtensions.class)
-
-    type_system.extendClass('java.util.concurrent.locks.Lock', LockExtensions.class)
-    type_system.extendClass('java.util.regex.Matcher', MatcherExtensions.class)
-
-    type_system.extendClass('int', IntExtensions.class)
-    type_system.extendClass('byte', NumberExtensions.class)
-    type_system.extendClass('short', NumberExtensions.class)
-    type_system.extendClass('int', NumberExtensions.class)
-    type_system.extendClass('long', NumberExtensions.class)
-    type_system.extendClass('float', NumberExtensions.class)
-    type_system.extendClass('double', NumberExtensions.class)
+    if self.builtins_enabled 
+      type_system.extendClass('java.util.Collection', Class.forName("org.mirah.builtins.CollectionExtensions"))
```

Comment:
If we're using Class.forName here, I think we should catch the lookup failure and re-raise with a better message.